src/plugins/dbg/quictypes.h

```diff
@@ -931,8 +931,8 @@ struct SentPacketMetadata : Struct {
         return ReadType<UINT64>("PacketNumber");
     }
 
-    UINT32 SentTime() {
-        return ReadType<UINT32>("SentTime"); // Microseconds
+    UINT64 SentTime() {
+        return ReadType<UINT64>("SentTime"); // Microseconds
```

Comment:
What if a new debug plugin is used to debug old, 32-bit time structures? Will this handle it gracefully?

Reply:
No. We never gracefully handle this in our debugger plugin

Reply:
We need to use the debugger plugin from the same MsQuic build. good to know. Should document that